desktop/cmp/rest/impl/RestForm.js

```diff
@@ -22,14 +24,14 @@ export const restForm = hoistCmp.factory({
     model: uses(RestFormModel),
     className: 'xh-rest-form',
 
-    render({model, className}) {
+    render({model, className, formClassName}) {
```

Comment:
you can pass the class name you are looking to pass to this implementation component just using the className argument.